src/Octokit.Webhooks/Events/WorkflowJobEvent.cs

```diff
@@ -7,4 +7,7 @@ public abstract record WorkflowJobEvent : WebhookEvent
 {
     [JsonPropertyName("workflow_job")]
     public Models.WorkflowJobEvent.WorkflowJob WorkflowJob { get; init; } = null!;
+
+    [JsonPropertyName("deployment")]
+    public Models.DeploymentEvent.Deployment? Deployment { get; init; }
```

Comment:
Are there specific `WorkflowJobEvent` actions that have this as non-null? Or could it appear on any action?

Reply:
From my understanding, `WorkflowJobEvent` actions that have an `environment` associated with them will include a `deployment` node, whereas all other jobs that don't have an `environment` will not include this property.

Ex: this kind of job **will** have a `deployment` node

```yaml
jobs:
  test:
    runs-on: ubuntu-latest
    environment: Test
    steps:
      - run: echo "Hello World"
```
 
If the `environment` node here we're left off, then the `deployment` property would be null.

Reply:
To clarify, the presence of the `deployment` property is not dependent on the action type, this property will appear on all of the action types (queued, in progress, completed and waiting). It just depends on whether the job has an `environment` attached to it.

Reply:
Thanks for the explanation. This looks great then!